content/_guides/cloaks.md

```diff
@@ -38,10 +38,9 @@ effect immediately.
 
 ## Project or community cloaks
 
-If you are involved with a well-known project or community, you may request a
-cloak indicating your affiliation. For help with this, ask someone associated
-with your project or community who can coordinate with Libera.Chat to
-administer cloaks.
+If you are involved with a registered [project](https://libera.chat/chanreg#project-registration) or [community](https://libera.chat/chanreg#community-registration), you may request a
+cloak indicating your affiliation. For help with this, ask a group
+contact for your project or community who can request cloaks.
```

Comment:
I *think* there should be a comma after community too

Reply:
I debated. As a restrictive appositive, I don't think it calls for a comma, and I think I am going for restrictive appositive. Perhaps not all group contacts can request a cloak, in which case the procedure is to ask specifically a member of that subset of group contacts who can request cloaks. But it does work the other way, if we're just saying to ask anyone who can request a cloak, which would be any group contact, in which case, comma. Happy to go with the group consensus either way.

Reply:
I'm in no way an expert on the topic. I simply thought it seemed like one should go there :)

Reply:
Well, I never miss a chance to overthink it!

Reply:
fixed it by reworking the sentence anyway. thanks!